File: ntss-win/FNSICloudConvert/FNSICloudConvertServer/src/main/java/com/fnsi/cloudconverter/migration/mongo/MongoCollectionConfig.java
package com.fnsi.cloudconverter.migration.mongo;

import lombok.Data;

/**
 * Mongo ダンプ対象コレクション設定 (mongo_dump_config.yaml の 1 エントリ)
 */
@Data
public class MongoCollectionConfig {
    /** コレクション名 */
    private String  name;
    /** ダンプ対象か */
    private boolean dump = true;
    /** 施設フィルターフィールド名（null の場合は全件） */
    private String  filterField;
}

File: Source/ntss-win/FNSICloudConvert/FNSICloudConvertClient/FNSICloudConvertClient/Models/MongoCollectionConfig.cs
namespace FNSICloudConvertClient.Models
{
    /// <summary>
    /// mongo_dump_config.yaml の 1 コレクション定義
    /// </summary>
    internal class MongoCollectionConfig
    {
        /// <summary>コレクション名</summary>
        public string Name { get; set; }

        /// <summary>ダンプ対象かどうか（false = スキップ）</summary>
        public bool Dump { get; set; }

        /// <summary>
        /// 施設フィルターに使用するフィールド名（null = 全件対象）
        /// 例: "facility_cd"
        /// </summary>
        public string FilterField { get; set; }
    }
}
